### ISSUE TITLE
Question: future / alternatives

### ISSUE THREAD

**Issue by lucienimmink** [winjs/winjs#1631]
Reading "this also means that we're not planning a new feature release"; what does Microsoft offer/recommend html/css/js developers to create UWP apps in the future?

**Comment by Odonno** [comment]
That's a good question. Currently, I hope for a complete new and light library that contains exclusively the UI controls of Windows 10 (without the winjs implementation).

To me, winstrap (https://github.com/winjs/winstrap) looks like a good alternative but it lacks some UI controls (like the splitview).

**Comment by TonyRAnderson** [comment]
Seeing how React Native and Ionic 2 have both announced full Windows 10 support, I'm guessing they're focused on having the larger third-party frameworks fill that gap. Also NativeScript is in early stages of porting Windows 10 runtime. 

http://blog.ionic.io/announcing-windows-support-in-ionic-2/
https://blogs.windows.com/buildingapps/2016/04/13/react-native-on-the-universal-windows-platform/

**Comment by quanglam2807** [comment]
I have the same concern.

**Comment by hubert-associates** [comment]
Silverlight, again!? .NET soon?

**Comment by diegosa06** [comment]
If this project is maintenance mode, why am I not seeing any bug fixes?

**Comment by quanglam2807** [comment]
+1

**Comment by nazrdogan** [comment]
+1

**Comment by dpierangeli** [comment]
+1

**Comment by Loginstudio** [comment]
+1

**Comment by Odonno** [comment]
A CSS/JS UI Library for Fluent design?

**Comment by Calliostro** [comment]
I think WinJS is dead. I don't think, there will be any update for the Fluent Design. And I would not recommend spending time for WinJS any longer.

**Comment by quanglam2807** [comment]
@Calliostro I agree. It's dead. If you go to https://docs.microsoft.com, Microsoft also no longer provides Javascript documentation for UWP. I think they now focus on C# & React Native.

**Comment by nazrdogan** [comment]
@Odonno I just found this yesterday https://github.com/myxvisual/react-uwp

**Comment by Loginstudio** [comment]
@Calliostro They are making another mistake. Now who guarantees the developers that even React Native or XAML will not abandoned suddenly?

**Comment by Odonno** [comment]
@Loginstudio I think too. But it's too soon to judge. The team are focused on XAML first, and maybe it will spread to other languages then.

@nazrdogan I'll take a look. But that would be f*ckingly amazing!

**Comment by ap3rus** [comment]
@nazrdogan from the first sight react-uwp looks amazing! Going to give it a deeper look.
